scene/main/viewport.cpp

```diff
@@ -67,7 +67,7 @@ void ViewportTexture::setup_local_to_scene() {
 	}
 
 	Node *vpn = local_scene->get_node(path);
-	ERR_FAIL_COND_MSG(!vpn, "ViewportTexture: Path to node is invalid.");
+	ERR_FAIL_COND_MSG(!vpn, "ViewportTexture: Path to node is invalid. If the path seems valid, move the node using the texture after the viewport.");
```

Comment:
By "after", does you mean "below" in the scene tree? We should try to use wording that's as explicit as possible.